src/ui/components/Header.js

```diff
@@ -42,13 +43,20 @@ class Header extends React.Component {
   }
 
   render() {
-    const { loading } = this.props;
+    const { loading, uploading } = this.props;
     const isLoaded = loading == 100;
+    const upload = uploading?.totalMB
+      ? `${(uploading?.uploadedMB / uploading?.totalMB) * 100}%`
+      : `${uploading?.uploadedMB}MB`;
+
     return (
       <>
         <div id="header">
           <div className="logo"></div>
+          {/* NOTE: Should we use the app state to set this status rather than relying on `element.innerText`? */}
```

Comment:
lets do the same thing with status as well

Reply:
Thoughts ont the interface for the status type above?